test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] I think you created this file before creating the .gitignore file , kindly delete it manually since it shouldn't be in your repo as mentioned in the requirements